html/changelogs/AI_ROUGE!!!.yml

```diff
@@ -0,0 +1,50 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.  
+author: SonicGotNuked
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: ----MALF UPDATES----
+  - rscadd: "Borg Combat Module now unlocks with synthetic domination. It only activates on all the current borgs, not the later ones."
+  - rscdel: "Removes code delta from elite encryption hack"
+  - rscadd: "Replaces code delta with code yellow"
+  - tweak: "Borgs come out of the robot transformer hacked"
+  - rscdel: "Combat Borgs no longer have a laser cannon"
+  - rscadd: "Seperates traitor borg powers into a seperate verbs tab for you to actually know about"
+  - admin: "Autoborger now tells admins that it made a borg"
+  - rscadd: "You shouldn't know what the Autoborger is at a glance. It has a more descriptive description"
+  - rscadd: "It now tells players that they are lobotamized"
```

Comment:
How ironic. He could save the autoborger from lobotomized typos, but could not save himself.